src/Mod/TechDraw/TechDrawTools/TDToolsMovers.py

```diff
@@ -0,0 +1,77 @@
+# ***************************************************************************
+# *   Copyright (c) 2022 - Wanderer Fan <wandererfan@gmail.com>             *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+"""Provides view moving functions for TD Tools."""
+
+import FreeCAD as App
+import FreeCADGui as Gui
+import TechDraw
+import os
+
+#move a simple view and its dependents (dimensions, balloons, etc) from p1 to p2
+def simpleViewMove(v, p1, p2, copy):
+    deps = [x for x in v.InList if x.isDerivedFrom("TechDraw::DrawView")]
+    if not copy:
+        for d in deps:
+            rc = p1.removeView(d)
+        rc = p1.removeView(v)
```

Comment:
What is the purpose of assigning `rc` the return value of the `removeView` function??